external/jacoco/README.md

```diff
@@ -0,0 +1,22 @@
+Running External Jacoco tests locally
```

Comment:
Please add following part before this `Running External Jacoco tests locally`

# External Jacoco Tests

Jacoco tests are part of the external third-party application tests that help verify that the Adoptium binaries are good, by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries.  For each application, we choose to run a selection of their functional tests.
Jacoco test material is pulled from the [Jacoco](https://github.com/jacoco/jacoco.git) repository.